phpBB/phpbb/db/doctrine/oracle_platform.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\doctrine;
+
+use Doctrine\DBAL\Platforms\OraclePlatform;
+
+/**
+ * Oracle specific schema restrictions for BC.
+ */
+class oracle_platform extends OraclePlatform
+{
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getVarcharTypeDeclarationSQL(array $column)
```

Comment:
```suggestion
	public function getVarcharTypeDeclarationSQL(array $column): string
```

Reply:
Does that work? shouldn't you have the same type decls as the parent?